.github/workflows/tests.yml

```
@@ -16,11 +16,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 21
+      - name: Set up JDK 25
         uses: actions/setup-java@v3
```

actionlint (1.7.9)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)